### PR TITLE
fix(release-pipeline): Adding the new infra needed to run auth tests …

### PR DESCRIPTION
**Pull request by vinagesh**
…in nuget test pipeline

## Checklist
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-iot-sdk-csharp/blob/master/.github/CONTRIBUTING.md).
- [ ] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- [ ] This pull-request is submitted against the `master` branch.

### CODE REVIEW

File: vsts/test-release-nuget.yaml [hunk 1]
```diff
@@ -86,6 +86,11 @@ jobs:
           LA_AAD_APP_ID: $(LA-AAD-APP-ID)
           LA_AAD_APP_CERT_BASE64: $(LA-AAD-APP-CERT-BASE64)
           LA_WORKSPACE_ID: $(LA-WORKSPACE-ID)
+          IOTHUB_X509_CHAIN_DEVICE_NAME: $(IOTHUB-X509-CHAIN-DEVICE-NAME)
+          HUB_CHAIN_DEVICE_PFX_CERTIFICATE: $(HUB-CHAIN-DEVICE-PFX-CERTIFICATE)
```
Comment (azabbasi) [inline vsts/test-release-nuget.yaml:90]: So it looks like they are both targetting the same infra environment variable group, it should suffice I think ! Thanks Sindhu.

**Review by azabbasi — APPROVED** [review]
LGTM

**Review by abhipsaMisra — APPROVED** [review]

File: vsts/test-release-nuget.yaml (hunk 1 above)
Comment (vinagesh) [reply vsts/test-release-nuget.yaml:90]: yes this should be sufficient

**Pull request merged**